cloud_build/verify_provenance.sh

```diff
@@ -8,7 +8,7 @@
 # legitimate, then the script will exit with a non-zero exit code.
 PROVENANCE_PATH=$1
 BUILDER_ID=https://cloudbuild.googleapis.com/GoogleHostedWorker@v0.3
-SOURCE_URI=https://github.com/flutter/cocoon
+SOURCE_URI=https://git+https://github.com/flutter/cocoon
```

Comment:
does the https:// prefix need to be removed?

Reply:
No, for some reason cloud build is adding that weird URI to the provenance.